src/transaction/log_replication_atomic.cpp

```diff
@@ -239,8 +239,14 @@ namespace cublog
 	    const LOG_REC_SCHEMA_MODIFICATION_LOCK log_rec =
 		    m_redo_context.m_reader.reinterpret_copy_and_add_align<LOG_REC_SCHEMA_MODIFICATION_LOCK> ();
 
+	    if (is_already_locked_for_ddl (header.trid, &log_rec.classoid))
```

Comment:
```suggestion
	    if (is_locked_for_ddl (header.trid, &log_rec.classoid))
```
 I don't think "already" is needed. At your choice.

Reply:
OK, I will change the name